### PR TITLE
Feature/summary/establish technical resources

### PR DESCRIPTION
**Pull request by larfeq**
- [x] New feature
- [ ] Bug fix
- [ ] High impact

**Work description:**

This PR is for establishing Summary Notification Web API resource for further development. It contains health endpoints for Kubernetes and swagger.

_Expected result:_ New web API resources established in PR environment

Link to workitem: [#51117](https://statoil-proview.visualstudio.com/Fusion%20Resource%20Allocation/_workitems/edit/51117)
 
**Test description:**
- [x] Can be tested
- [ ] Automatic tests created / updated
- [ ] Local tests are passing

 
**Checklist:**
- [x] Considered automated tests
- [ ] Considered updating specification / documentation
- [ ] Considered work items 
- [ ] Considered security
- [x] Performed developer testing
- [x] Checklist finalized / ready for review

### CODE REVIEW

File: src/Fusion.Summary.Api/Fusion.Summary.Api.http [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@Fusion.Summary.Api_HostAddress = http://localhost:5229
+
+GET {{Fusion.Summary.Api_HostAddress}}/weatherforecast/
```
Comment (aleklundeq) [inline src/Fusion.Summary.Api/Fusion.Summary.Api.http:3]: What is this. Is this necessarry for this pr?

Comment (larfeq) [reply src/Fusion.Summary.Api/Fusion.Summary.Api.http]: Its a test, built in, came with the project. Removed it.

**Review by aleklundeq — APPROVED** [review]

File: src/Fusion.Summary.Api/Deployment/k8s/deployment-pr-env.yml [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+          value: "{{instrumentationKey}}"
+        - name: KEYVAULT_URL
+          value: "{{envKeyVaultUrl}}"
+        - name: ENVNAME
+          value: "{{ENVNAME}}"
+        - name: FUSION_ENVIRONMENT
+          value: "{{fusionEnvironment}}"
+        - name: AzureAd__ClientId
+          value: "{{clientId}}"
+        - name: ConnectionStrings__ResourcesDbContext
```
Comment (aleklundeq) [inline src/Fusion.Summary.Api/Deployment/k8s/deployment-pr-env.yml:67]: Is this something that we should remove for this pr?

Comment (larfeq) [reply src/Fusion.Summary.Api/Deployment/k8s/deployment-pr-env.yml]: Fixd!

**Pull request merged**